core/app/models/spree/import_mapping.rb

```
@@ -18,16 +18,29 @@ class ImportMapping < Spree.base_class
       import.required_fields.include?(schema_field)
     end
 
+    # Returns true if the mapping has a file column
+    # @return [Boolean]
     def mapped?
       file_column.present?
     end
 
     def try_to_auto_assign_file_column(csv_headers)
-      self.file_column = csv_headers.find { |header| header.parameterize.downcase.strip == schema_field.parameterize.downcase.strip }
+      self.file_column = csv_headers.find { |header| header.parameterize.underscore.downcase.strip == schema_field.parameterize.underscore.downcase.strip }
```

RuboCop (1.81.6)

[convention] Line is too long. [155/150]

(Layout/LineLength)